{{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/tools/make_artifacts.py

```diff
@@ -97,7 +96,7 @@ def build_artifacts(
     check_for_existing(output_dir, location, append, replace_keys)
 
     if location in metadata.LOCATIONS:
-            build_single(location, output_dir, replace_keys)
+            build_single(loc, output_dir, replace_keys)
```

Comment:
I think this should be location instead of loc.

Reply:
Unclear how this worked without that tbqh.